src/fillTank.test.js

```diff
@@ -1,11 +1,84 @@
-'use strict';
+"use strict";
 
-describe('fillTank', () => {
-  // const { fillTank } = require('./fillTank');
 
-  it('should ', () => {
+describe('fillTank', () => {
+  const { fillTank } = require('./fillTank');
+  const customer = {
+    money: 3000,
+    vehicle: {
+      maxTankCapacity: 40,
+      fuelRemains: 8,
+    },
+  };
+  const fuelPrice = 4.80;
 
+  it('should be declared', () => {
+    expect(fillTank).toBeInstanceOf(Function);
   });
 
   // write tests here
+  it('should round the poured amount to the tenth part', () => {
+    fillTank(customer, fuelPrice, 20.78);
```

Comment:
The test 'should round the poured amount to the tenth part' does not mock the function Math.round. Thus, this test may fail due to the default behavior of Math.round. It's better to use mocking in order to control the function's output.